elements/pfe-accordion/package.json

```diff
@@ -8,25 +8,38 @@
   "publishConfig": {
     "access": "public"
   },
-  "description": "Accordion element for PFElements",
+  "description": "Accordion for PatternFly Elements",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/patternfly/patternfly-elements/tree/v1.0.0-prerelease.10/elements/pfe-accordion"
```

Comment:
Per the [npm docs](https://docs.npmjs.com/files/package.json#repository), the repository url field needs to point to a git repo, something you can run `git clone` on, so this URL which points to a subdirectory within a tag doesn't meet that requirement.  Fortunately, the docs offer a solution!  There is an optional `directory` field to specify a subdirectory within the git repo where the given element lives.

```json
"repository": {
  "type" : "git",
  "url" : "https://github.com/patternfly/patternfly-elements.git",
  "directory": "elements/pfe-accordion"
}
```

We can also, optionally, use the `github:patternfly/patternfly-elements` shorthand for the url field.

```json
"repository": {
  "type" : "git",
  "url" : "github:patternfly/patternfly-elements",
  "directory": "elements/pfe-accordion"
}
```

Reply:
I love that shorthand.